Java/libraries/recognizers-text-date-time/src/main/java/com/microsoft/recognizers/text/datetime/parsers/BaseDateParser.java

```diff
@@ -177,8 +177,7 @@ private DateTimeResolutionResult parseImplicitDate(String text, LocalDateTime re
 
         if (exactMatch.getSuccess()) {
             int swift = getSwiftDay(exactMatch.getMatch().get().value);
-
-            LocalDateTime value = referenceDate.plusDays(swift);
+            LocalDateTime value = referenceDate.toLocalDate().atStartOfDay().plusDays(swift);
```

Comment:
Is this really the same behaviour as in .NET? Any usage of toLocal() always make me nervous of causing timezone issues.

Reply:
toLocalDate() just return date information.

public LocalDate toLocalDate() {
    return date;
} 
i find this also used in resolve End Of Day

    private DateTimeResolutionResult resolveEndOfDay(String timexPrefix, LocalDateTime futureDate, LocalDateTime pastDate) {
        String timex = String.format("%sT23:59:59", timexPrefix);
        LocalDateTime futureValue = LocalDateTime.of(futureDate.toLocalDate(), LocalTime.MIDNIGHT).plusDays(1).minusSeconds(1);
        LocalDateTime pastValue = LocalDateTime.of(pastDate.toLocalDate(), LocalTime.MIDNIGHT).plusDays(1).minusSeconds(1);

        DateTimeResolutionResult result = new DateTimeResolutionResult();
        result.setTimex(timex);
        result.setFutureValue(futureValue);
        result.setPastValue(pastValue);
        result.setSuccess(true);

        return result;
    }

Reply:
OK.